server/configuration_service.py

```diff
@@ -0,0 +1,29 @@
+import asyncio
+
+from .config import config
+from .core import Service
+from .decorators import with_logger
+
+
+@with_logger
+class ConfigurationService(Service):
+    def __init__(self) -> None:
+        self._logger.info("Configuration service created.")
+        self._store = config
+        self._task = None
+
+    async def initialize(self) -> None:
+        self._task = asyncio.create_task(self._worker_loop())
+        self._logger.info("Configuration service started.")
+
+    async def _worker_loop(self) -> None:
+        while True:
+            self._logger.info("Refreshing configuration variables")
```

Comment:
I think I would put this on `debug` log level and instead log when changes are detected. So for instance if I change `QUEUE_POP_TIME_MAX` it would log something like:
```
New value for QUEUE_POP_TIME_MAX: 180 -> 200
```